csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -14,11 +14,50 @@
 #include <scheduler/normalization_utils.h>
 #include <scheduler/registry_utils.h>
 #include <scheduler/runtime_info.h>
+#include <scheduler/utils.h>
 
 #include <ATen/cuda/CUDAContext.h>
 
 namespace nvfuser {
 namespace {
+
+// Prioritize warp specialized approach if possible
+bool preferWarpSpecialized(
+    Fusion* fusion,
+    int64_t total_iteration_numel,
+    int64_t n_inner_reduction_tvs) {
+  // False, for pre-Blackwell GPUs
+  if (at::cuda::getCurrentDeviceProperties()->major < 10) {
+    return false;
+  }
+  // False, if any of the inputs is not concretized
+  // TODO: extend to support dynamic inputs, warp specialization requires
+  // static CTA size
+  auto inp_tvs = ir_utils::filterByType<TensorView>(fusion->inputs());
+  if (std::any_of(inp_tvs.begin(), inp_tvs.end(), [](TensorView* tv) {
+        return scheduler_utils::isConcreteTensor(tv);
```

Comment:
The logic looks wrong to me.
```suggestion
        return !scheduler_utils::isConcreteTensor(tv);
```

Reply:
Changed to `isSymbolicTensor`, so no need to change the logic here.